docs/installation.md

```diff
@@ -86,6 +86,10 @@ You can optionally install a different version with Homebrew.
 
 #### Elasticsearch
 
+!!! note
+  __These instructions are deprecated since Elasticsearch 1.7
+  is no longer supported by `brew`__
```

Comment:
I think this is sufficient right now. @rosskarchner is working on ES 2.4 compatibility but we're not there yet (and this is a subtle reminder that we'll need to update these instructions once he's done).